src/js/constants/NameMappings.js

```diff
@@ -0,0 +1,13 @@
+var AppStatus = require("../constants/AppStatus");
+
+const NameMappings = {
+  statusNameMapping: {
```

Comment:
Why not just export the top-level object directly like we do in `AppListViewTypes` etc? Do you see this AppStatus growing to support other objects?